apps/client/src/features/rundown/event-editor/composite/EventColorPicker.tsx

```diff
@@ -0,0 +1,35 @@
+import { debounceWithValue } from '../../../../common/utils/debounce';
+import PopoverPicker from '../../../../common/components/input/popover-picker/PopoverPicker';
+import style from './EventColorPicker.module.scss';
+import { ReactNode, useCallback } from 'react';
+
+interface EventColorPickerProps {
+  name: 'colour';
+  value: string;
+  icon: ReactNode;
+  handleChange: (newValue: 'colour', name: string) => void;
+}
+
+export default function EventColorPicker(props: EventColorPickerProps) {
```

Comment:
nice! I really appreciate the work you put into following project conventions

suggestion: I wonder if this would be better called `SwatchPicker` to signify its relationship to the other Swatch elements? We could also collocate this in the `components/input/colour-input directory`